src/rfc/detail/rules.cpp

```diff
@@ -397,6 +398,75 @@ remove_obs_fold(
     }
 }
 
+//------------------------------------------------
+
+system::result<hex_rule_t::value_type>
+hex_rule_t::
+parse(
+    char const*& it,
+    char const* end) const noexcept
+{
+    value_type v;
+
+    constexpr
+    auto const max =
```

Comment:
As a general rule, never use the name `max` if it can be avoided, as it is sometimes a macro which leads to compile errors on specific toolchains.

Reply:
Also, the same rule for `min` :)